sbn/codingconv/CodingConventions.md

```diff
@@ -147,7 +147,7 @@ possible name collisions or ambiguities.
   `double const F = G * m * M / (d*d);` is acceptable, but 
   `double const F = G * m1 * m2 / (d*d);` should still be preferred.
  
-**[E]** Starting private data members with `f` and use CamelCase is **encouraged** (e.g. `double fTrackLength`).
+**[E]** Starting private data members with `f` and use CamelCase is **encouraged** (e.g. `double fTrackLength`). Agree, but can we spell this out more / ban starting non-members with `f`. Lots of people seem to be confused about this.
```

Comment:
How about:
```suggestion
**[E]** Names starting with `f` and using CamelCase (e.g. `double fTrackLength`) are **encouraged** for private data members, and only for them. Conversely, the use of the trailing `f` for local variables is **discouraged**.
```

Reply:
"leading", not "trailing" `f`, and I don't think you want to discourage camel case for local variables.
Maybe this is clearer with examples: `ClassName`, `FunctionName()`, `fMemberVar`, `localVar` etc

Reply:
I feel like we need a example table (plus the load of examples throughout the test, if consistent).
I'll set to this test and schedule a table.
```suggestion
**[E]** Names starting with `f` and using CamelCase (e.g. `double fTrackLength`) are **encouraged** for private data members, and only for them. Conversely, the use of a leading `f` for local variables is **discouraged**.
```

Reply:
you still write that camel case is only for private members. Are you calling `thisStyle` for local variables by a different name? We might need to clarify the jargon in a table / with examples as well.

Reply:
_I_ call `thisStile` drinking camelcase and `ThisStyle` staring camelcase. I am sure that nomenclature is universal and internationally acclaimed.
But to your point, I get it: you prefer the full rule spelled out.
```
**[E]** Private data member names are **encouraged** to start with `f` and use CamelCase
   (e.g. `double fTrackLength`). This pattern should be reserved exclusively for such private data member.
  Conversely, public data members and local variables should instead follow the more general guideline
  expressed above (i.e. simple camelCase, e.g. `trackLength`).
```